interfaces/naming.go

```diff
@@ -44,3 +46,17 @@ func WrapperNameForApp(snapName, appName string) string {
 func SecurityTagForApp(snapName, appName string) string {
 	return fmt.Sprintf("%s.snap", WrapperNameForApp(snapName, appName))
 }
+
+// SecurityTag returns application-specific security tag.
```

Comment:
SecurityTagForApp and everything else that doesn't use snap.Info is deprecated. It gets removed with the last backend.